packages/playground/src/components/webthree-connector/webthree-connector.tsx

```diff
@@ -75,16 +85,13 @@ export class Web3Connector {
       user: {
         ...this.accountState.user,
         ethAddress: this.getCurrentAddress()
-      },
-      balance: 0,
-      accountBalance: 0
+      }
     });
 
-    const provider = new Web3Provider(this.getProvider());
     this.walletState.updateWalletConnection!({
       ...(await this.getCurrentWalletState()),
-      provider,
-      signer: provider.getSigner(),
+      provider: this.getProvider(),
```

Comment:
@snario This doesn't seem necessary.

Reply:
It is necessary, this is the only place it gets set